Mathlib/Combinatorics/SimpleGraph/StronglyRegular.lean

```diff
@@ -176,4 +175,35 @@ theorem IsSRGWith.compl (h : G.IsSRGWith n k ℓ μ) :
 set_option linter.uppercaseLean3 false in
 #align simple_graph.is_SRG_with.compl SimpleGraph.IsSRGWith.compl
 
+/-- The parameters of a strongly regular graph with at least one vertex satisfy
+`k * (k - ℓ - 1) = (n - k - 1) * μ`. -/
+theorem IsSRGWith.param_eq (h : G.IsSRGWith n k ℓ μ) (hn : 0 < n) :
+    k * (k - ℓ - 1) = (n - k - 1) * μ := by
+  rw [← h.card, Fintype.card_pos_iff] at hn
+  have v := Classical.choice hn
```

Comment:
```suggestion
  obtain \<v\> := hn
```
avoids choice here

(sorry, you'll have to type the unicode)

Reply:
Done.